Sources/MenuContainerViewController.swift

```diff
@@ -55,6 +55,19 @@ open class MenuContainerViewController: UIViewController {
         screenEdgePanRecognizer.edges = .left
         self.view.addGestureRecognizer(screenEdgePanRecognizer)
     }
+
+    override open func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        coordinator.animate(alongsideTransition: { _ in
+            let viewBounds = CGRect(x:0, y:0, width:size.width, height:size.height)
+            let viewCenter = CGPoint(x:size.width/2, y:size.height/2)
```

Comment:
it is not critical but no any reason to insert this preparation code to the animation block